pkg/shp/cmd/build/run_test.go

```diff
@@ -124,7 +124,7 @@ func TestStartBuildRunFollowLog(t *testing.T) {
 		}
 		shpclientset.PrependReactor("create", "buildruns", createReactorFunc)
 		// need this reactor to handle returning our test BuildRun with whatever updates we make based on the various
-		// test bools that result in spec.state or deletion timestamp updates
+		// test boils that result in spec.state or deletion timestamp updates
```

Comment:
I agree bools isn't right, but not sure boils is either

maybe just remove both bools and boils and go with `tests that result in spec.state or deletion timestamp updates `

Reply:
Oops, sorry. The I end up using the spellcheck quick fix accidentally, so I'm saving it for a future PR, please.

Reply:
Oops, sorry! I accidentally activated the spellchecker quick fix on the word, so let me revert this change and save this information for a later interaction, please.

Reply:
+1